src/test/scala/chiselTests/BundleLiteralSpec.scala

```diff
@@ -73,4 +93,24 @@ class BundleLiteralSpec extends ChiselFlatSpec {
       stop()
     } }
   }
+
+  "bundles inside of bundles" should "work in RTL" in {
+    assertTesterPasses{ new BasicTester{
+      val bundleLit = (new MyOuterBundle).Lit(42.U, true.B, 22.U)
+      chisel3.assert(bundleLit.c === 22.U)
+      // TODO!!!
+      // chisel3.assert(bundleLit.d.a === 42.U)
```

Comment:
Why don't these work? Why can't support for these be implemented in this PR, instead of having dead code?

Reply:
I think it's dying in firrtl, still need to track down exactly why. 1 level of bundles is ok, apparently nesting is not.

Reply:
These work now